lib/internal/Magento/Framework/Console/CommandLoader/Aggregate.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+
+namespace Magento\Framework\Console\CommandLoader;
+
+use Symfony\Component\Console\CommandLoader\CommandLoaderInterface;
+use Symfony\Component\Console\Exception\CommandNotFoundException;
+
+/**
+ * Class Aggregate has a list of command loaders, which can be extended via DI configuration.
+ */
+class Aggregate implements CommandLoaderInterface
+{
+    /** @var CommandLoaderInterface[] */
+    private $commandLoaders;
+
+    public function __construct($commandLoaders = [])
+    {
+        $this->commandLoaders = $commandLoaders;
+    }
+
+    /**
+     * @param string $name
+     * @return \Symfony\Component\Console\Command\Command
+     */
+    public function get($name)
```

Comment:
Introduce strict type
```suggestion
    public function get(string $name)
```

Reply:
I don't believe this is possible–the [interface being implemented does not use strict types](https://github.com/symfony/console/blob/4.4/CommandLoader/CommandLoaderInterface.php#L31).

Here's an example in action: https://3v4l.org/gml83

Reply:
But seems like return type we still could define
```suggestion
    public function get($name): \Symfony\Component\Console\Command\Command
```

Here is an example in action: https://3v4l.org/nQTa0